ChangeLog

```diff
@@ -15,6 +15,11 @@ Release date: TBA
 
   Closes #4982
 
+* Introduced primer tests and a configuration tests framework. The helper classes available in
+  ``pylint/testutil/`` are still unstable and might be modified in the near future.
+
+  Closes #4412 #5287
```

Comment:
Is the issue number correct? #5287

Reply:
Yes this is the configuration's test framework issue. It will also save us some headache to say it's unstable for now.